### ISSUE TITLE
Failed to update FTP server. Check wrapper log for more info!

### ISSUE THREAD

**Issue by B3H1NDu** [TunkDesign/NeMa#23]
Logs: http://hastebin.com/yezubicula.vbs
The problem seems to be: 
```
File "/TunkDesign/Build/NeMa/build/NeMa/out00-PYZ.pyz/scripts.filemanager", line 142, in stop
RuntimeError: Server not started yet
```
I've tried to look for the files but no luck.

I am running Ubuntu 16.04 LTS (not server edition)

I have tried everything, restarting, using different browsers and computers to access the webpanel and ssh from but nothing seems to be working.

I hope you can help me :)

**Comment by dedo1911** [comment]
I'm having the same issue, on the same Ubuntu version.

**Comment by dedo1911** [comment]
I've reinstalled and changed to CentOS 7 and problem is gone.

**Comment by B3H1NDu** [comment]
Yea, there is a required package which is missing probably.

**Comment by TunkDesign** [comment]
I haven't tried running it on Ubuntu for quite a while, so I will do some debugging this weekend to see if anything is missing!

**Comment by B3H1NDu** [comment]
Any progress? Anything I can do to help?

**Comment by TunkDesign** [comment]
@B3H1NDu I am really sorry! I know I said I would look at it 10 days ago, but I just haven't had the time. I am doing during my free time, which is kind of limited at the moment due to family issues. (I won't go into detail)
Something happened that weekend, and I haven't gotten around to do anything. I can't promise anything, but I'll take a look at it as soon as possible. **Really** sorry!

**Comment by B3H1NDu** [comment]
No problem man! Hope your family stuff gets resolved and keep up the good work!

**Comment by TunkDesign** [comment]
@B3H1NDu I finally got some time to look at this and I found the issue. The FTP server needs to stop in order for NeManager to add a server, and to do that I tell it to stop before the adding, then start again when the server has been added. The issue here was that it wasn't turned on again because it failed somewhere during the server creation process. This should be fixed in the next release :)

Will update and close this issue when the next release is out :D
